### PR TITLE
[DM-3890]: Changed DM title to be title cased.

### PR DESCRIPTION
**Pull request by dafkk**

### CODE REVIEW

File: content/concepts/concepts-introduction-bundle/device-management.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Device management
+title: Device Management
```
Comment (BeateRixen) [inline content/concepts/concepts-introduction-bundle/device-management.md:2]:
```suggestion
title: Device management
```

File: content/concepts/concepts-introduction.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ sector:
 {{< product-c8y-iot >}} provides
 
 * Certified software libraries you can use to bring your remote assets into the cloud.
-* Device management, data visualization and remote control functionality through the web.
+* Device Management, data visualization and remote control functionality through the web.
```
Comment (BeateRixen) [inline content/concepts/concepts-introduction.md:16]:
```suggestion
* Device management, data visualization and remote control functionality through the web.
```

File: content/concepts/interfacing-devices.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ To interface these systems with {{< product-c8y-iot >}}, a driver software calle
 {{< c8y-admon-related >}}
 
 - [Getting started > Technical concepts > {{< product-c8y-iot >}}´s domain model](/concepts/domain-model) for understanding the data structures exchanged between agents and the {{< product-c8y-iot >}} core.
-- [Device management > Device integration](/device-integration) for understanding in detail how to develop agent software using the REST or MQTT protocols.
+- [Device Management > Device integration](/device-integration) for understanding in detail how to develop agent software using the REST or MQTT protocols.
```
Comment (BeateRixen) [inline content/concepts/interfacing-devices.md]:
```suggestion
- [Device management & connectivity > Device integration](/device-integration) for understanding in detail how to develop agent software using the REST or MQTT protocols.
```

File: content/device-integration/fragment-library-bundle/fragment-ibrary-introduction.md [hunk 4]
```diff
@@ -14,10 +14,10 @@ If you are interested in details on exposing the {{< product-c8y-iot >}}´s func
 
 Moreover, see [SmartREST](/smartrest/smartrest-two) for more information on SmartREST and a complete list of all SmartREST templates mentioned throughout the following sections.
 
-To start with device management, open the **All devices** tab in the **Devices** menu of the Device management application. Click on a device in the list to open the device details of this particular device. You will see various tabs and particular information on each of them.
+To start with device management, open the **All devices** tab in the **Devices** menu of the Device Management application. Click on a device in the list to open the device details of this particular device. You will see various tabs and particular information on each of them.
 
 {{< c8y-admon-info >}}
-For a detailed explanation of each tab and its related configuration via the UI, see also [Device management application > Viewing device details](/device-management-application/viewing-device-details).
+For a detailed explanation of each tab and its related configuration via the UI, see also [Device Management application > Viewing device details](/device-management-application/viewing-device-details).
```
Comment (BeateRixen) [inline content/device-integration/fragment-library-bundle/fragment-ibrary-introduction.md:20]:
```suggestion
For a detailed explanation of each tab and its related configuration via the UI, see also [Viewing device details](/device-management-application/viewing-device-details).
```

File: content/device-management-application/connectivity.md [hunk 5]
```diff
@@ -7,11 +7,11 @@ sector:
 ---
 
 {{< c8y-admon-related >}}
-* [Device management > Device integration > Fragment library > Connectivity](/device-integration/fragment-library/#connectivity) for details on the `c8y_Mobile` fragment used in managed objects.
+* [Device Management > Device integration > Fragment library > Connectivity](/device-integration/fragment-library/#connectivity) for details on the `c8y_Mobile` fragment used in managed objects.
```
Comment (BeateRixen) [inline content/device-management-application/connectivity.md]:
```suggestion
* [Device management & conectivity > Device integration > Fragment library > Connectivity](/device-integration/fragment-library/#connectivity) for details on the `c8y_Mobile` fragment used in managed objects.
```

File: content/device-management-application/managing-device-data.md [hunk 6]
```diff
@@ -10,12 +10,12 @@ sector:
 ---
 
 {{< c8y-admon-related >}}
-* [Device management > Device integration > Fragment library > Firmware](/device-integration/fragment-library/#firmware) for details on the `c8y_Firmware` fragment used in managed objects.
-* [Device management > Device integration > Fragment library > Software](/device-integration/fragment-library/#software) for details on legacy software management via the `c8y_SoftwareList` fragment, software updates via the `c8y_SoftwareUpdate` fragment and advanced software management via the `c8y_SupportedOperations` and `c8y_SupportedSoftwareTypes` fragments used in managed objects.
-* [Device management > Device integration > Fragment library > Configuration](/device-integration/fragment-library/#configuration) for details on text-based configuration, file-based configuration and typed file-based configuration.
+* [Device Management > Device integration > Fragment library > Firmware](/device-integration/fragment-library/#firmware) for details on the `c8y_Firmware` fragment used in managed objects.
```
Comment (BeateRixen) [inline content/device-management-application/managing-device-data.md]:
```suggestion
* [Device management & connectivity > Device integration > Fragment library > Firmware](/device-integration/fragment-library/#firmware) for details on the `c8y_Firmware` fragment used in managed objects.
```

File: content/device-management-application/managing-device-data.md [hunk 7]
```diff
@@ -10,12 +10,12 @@ sector:
...
+* [Device Management > Device integration > Fragment library > Software](/device-integration/fragment-library/#software) for details on legacy software management via the `c8y_SoftwareList` fragment, software updates via the `c8y_SoftwareUpdate` fragment and advanced software management via the `c8y_SupportedOperations` and `c8y_SupportedSoftwareTypes` fragments used in managed objects.
```
Comment (BeateRixen) [inline content/device-management-application/managing-device-data.md]:
```suggestion
* [Device management & connectivity > Device integration > Fragment library > Software](/device-integration/fragment-library/#software) for details on legacy software management via the `c8y_SoftwareList` fragment, software updates via the `c8y_SoftwareUpdate` fragment and advanced software management via the `c8y_SupportedOperations` and `c8y_SupportedSoftwareTypes` fragments used in managed objects.
```

File: content/device-management-application/managing-device-data.md [hunk 8]
```diff
@@ -10,12 +10,12 @@ sector:
...
+* [Device Management > Device integration > Fragment library > Configuration](/device-integration/fragment-library/#configuration) for details on text-based configuration, file-based configuration and typed file-based configuration.
```
Comment (BeateRixen) [inline content/device-management-application/managing-device-data.md]:
```suggestion
* [Device management & connectivity > Device integration > Fragment library > Configuration](/device-integration/fragment-library/#configuration) for details on text-based configuration, file-based configuration and typed file-based configuration.
```

File: content/device-management-application/managing-device-data.md [hunk 9]
```diff
@@ -10,12 +10,12 @@ sector:
...
 * The [device credentials API](https://{{< domain-c8y >}}/api/core/#tag/Device-credentials) for REST API methods concerning device credentials.
-* [Device management > Device integration > Fragment library > Device profile](/device-integration/fragment-library/#device-profile) for details on the `c8y_DeviceProfile` fragment used in managed objects.
-* [Device management > Device integration > Device integration using MQTT > Device certificates](/device-integration/mqtt/#device-certificates) for information on device certificates in the context of MQTT.
+* [Device Management > Device integration > Fragment library > Device profile](/device-integration/fragment-library/#device-profile) for details on the `c8y_DeviceProfile` fragment used in managed objects.
```
Comment (BeateRixen) [inline content/device-management-application/managing-device-data.md]:
```suggestion
* [Device management & connectivity > Device integration > Fragment library > Device profile](/device-integration/fragment-library/#device-profile) for details on the `c8y_DeviceProfile` fragment used in managed objects.
```

File: content/device-management-application/managing-device-data.md [hunk 10]
```diff
@@ -10,12 +10,12 @@ sector:
...
+* [Device Management > Device integration > Device integration using MQTT > Device certificates](/device-integration/mqtt/#device-certificates) for information on device certificates in the context of MQTT.
```
Comment (BeateRixen) [inline content/device-management-application/managing-device-data.md]:
```suggestion
* [Device management & connectivity > Device integration > Device integration using MQTT > Device certificates](/device-integration/mqtt/#device-certificates) for information on device certificates in the context of MQTT.
```

File: content/device-management-application/managing-device-types.md [hunk 11]
```diff
@@ -18,7 +18,7 @@ helpcontent:
 
 {{< c8y-admon-related >}}
 * [Getting started > Technical concepts > Interfacing devices](/concepts/interfacing-devices/) for information on the concepts relevant for interfacing IoT devices and other IoT-related data sources with {{< product-c8y-iot >}}.
-* [Device management > Protocol integration](/protocol-integration/) for a list of protocols, parameters and network connectivity options for devices.
+* [Device Management > Protocol integration](/protocol-integration/) for a list of protocols, parameters and network connectivity options for devices.
```
Comment (BeateRixen) [inline content/device-management-application/managing-device-types.md]:
```suggestion
* [Device management & connectivity > Protocol integration](/protocol-integration/) for a list of protocols, parameters and network connectivity options for devices.
```

File: content/device-management-application/monitoring-and-controlling-devices.md [hunk 12]
```diff
@@ -10,9 +10,9 @@ sector:
 ---
 
 {{< c8y-admon-related >}}
-* [Device management > Device integration > Fragment library > Device availability](/device-integration/fragment-library/#device-availability) for details on the `c8y_RequiredAvailability`, `c8y_UnavailabilityAlarm`, `c8y_Availability` and `c8y_Connection` fragments used in managed objects.
+* [Device Management > Device integration > Fragment library > Device availability](/device-integration/fragment-library/#device-availability) for details on the `c8y_RequiredAvailability`, `c8y_UnavailabilityAlarm`, `c8y_Availability` and `c8y_Connection` fragments used in managed objects.
```
Comment (BeateRixen) [inline content/device-management-application/monitoring-and-controlling-devices.md]:
```suggestion
* [Device management & connectivity > Device integration > Fragment library > Device availability](/device-integration/fragment-library/#device-availability) for details on the `c8y_RequiredAvailability`, `c8y_UnavailabilityAlarm`, `c8y_Availability` and `c8y_Connection` fragments used in managed objects.
```

File: content/device-management-application/monitoring-and-controlling-devices.md [hunk 13]
```diff
@@ -10,9 +10,9 @@ sector:
...
 * The [alarms API](https://{{< domain-c8y >}}/api/core/#tag/Alarms) for REST API methods concerning alarms.
-* [Device management > Device integration > Fragment library > Alarms](/device-integration/fragment-library/#alarms) for details on how to raise and clear alarms.
+* [Device Management > Device integration > Fragment library > Alarms](/device-integration/fragment-library/#alarms) for details on how to raise and clear alarms.
```
Comment (BeateRixen) [inline content/device-management-application/monitoring-and-controlling-devices.md]:
```suggestion
* [Device management & connectivity > Device integration > Fragment library > Alarms](/device-integration/fragment-library/#alarms) for details on how to raise and clear alarms.
```

File: content/device-management-application/monitoring-device-services.md [hunk 14]
```diff
@@ -8,10 +8,10 @@ sector:
 
 {{< c8y-admon-related >}}
 * The [alarms API](https://{{< domain-c8y >}}/api/core/#tag/Alarms) for REST API methods concerning alarms.
-* [Device management > Device integration > Fragment library > Alarms](/device-integration/fragment-library/#alarms) for details on how to raise and clear alarms.
+* [Device Management > Device integration > Fragment library > Alarms](/device-integration/fragment-library/#alarms) for details on how to raise and clear alarms.
```
Comment (BeateRixen) [inline content/device-management-application/monitoring-device-services.md]:
```suggestion
* [Device management & connectivity > Device integration > Fragment library > Alarms](/device-integration/fragment-library/#alarms) for details on how to raise and clear alarms.
```

File: content/device-integration/mqtt.md [hunk 15]
```diff
@@ -11,7 +11,7 @@ The MQTT implementation of {{< product-c8y-iot >}} provides the following benefi
 * Multi-tenancy support: A single endpoint serves multiple tenants.
 * Device identity management: Devices authenticate using device-specific credentials.
 * Device registration: Non-personalized devices can be deployed by pairing them with {{< product-c8y-iot >}} tenants.
-* Device management: Rich, pre-defined device management payload formats to enable out-of-the-box management of millions of devices.
+* Device Management: Rich, pre-defined device management payload formats to enable out-of-the-box management of millions of devices.
```
Comment (BeateRixen) [inline content/device-integration/mqtt.md:14]:
```suggestion
* Device management: Rich, pre-defined device management payload formats to enable out-of-the-box management of millions of devices.
```

File: content/device-management-application/viewing-device-details.md [hunk 16]
```diff
@@ -203,7 +203,7 @@ Only one Y-axis is rendered if the measurements belong to the same series.
 To see detailed information about the measured values, hover over the chart. A tooltip will be displayed with detailed information on the measurement next to your cursor (the tooltip will "snap" to the closest measurement).
 
 {{< c8y-admon-info >}}
-We recommend you to have max. 20 series per measurement for optimal performance and readability of a single graph in Device management (the graph displays all available series). If you need to display only a few series from the measurement, we advise you to use Data explorer in Cockpit where you can select series to be shown in the graph.
+We recommend you to have max. 20 series per measurement for optimal performance and readability of a single graph in Device Management (the graph displays all available series). If you need to display only a few series from the measurement, we advise you to use Data explorer in Cockpit where you can select series to be shown in the graph.
```
Comment (BeateRixen) [inline content/device-management-application/viewing-device-details.md:206]:
```suggestion
We recommend you to have max. 20 series per measurement for optimal performance and readability of a single graph in the Device Management application (the graph displays all available series). If you need to display only a few series from the measurement, we advise you to use Data explorer in Cockpit where you can select series to be shown in the graph.
```

File: content/glossary/p.md [hunk 17]
```diff
@@ -23,7 +23,7 @@ See also [General Availability](/glossary/g/#ga) and [Public Preview](/glossary/
 
 ### Product area {#product-area}
 
-In the change logs (or release notes for yearly releases respectively) changes can be filtered for various parameters one of which is the product area. The product area is a high-level functional area, for example, "Device Management & connectivity". 
+In the change logs (or release notes for yearly releases respectively) changes can be filtered for various parameters one of which is the product area. The product area is a high-level functional area, for example, "Device Management & Connectivity".
```
Comment (BeateRixen) [inline content/glossary/p.md:26]:
```suggestion
In the change logs (or release notes for yearly releases respectively) changes can be filtered for various parameters one of which is the product area. The product area is a high-level functional area, for example, "Device management & connectivity". 
```

File: content/protocol-integration/lwm2m-bundle/device-details.md [hunk 18]
```diff
@@ -5,7 +5,7 @@ layout: redirect
 ---
 
 {{< c8y-admon-info >}}
-In the Device management application, you can view all details of a device. The following details are specific to LWM2M devices. For information on general details refer to [Viewing device details](/device-management-application/viewing-device-details) in the Device management section.
+In the Device Management application, you can view all details of a device. The following details are specific to LWM2M devices. For information on general details refer to [Viewing device details](/device-management-application/viewing-device-details) in the Device Management section.
```
Comment (BeateRixen) [inline content/protocol-integration/lwm2m-bundle/device-details.md:8]:
```suggestion
In the Device Management application, you can view all details of a device. The following details are specific to LWM2M devices. For information on general details refer to [Viewing device details](/device-management-application/viewing-device-details).
```

File: content/sector/device_management/_index.md [hunk 19]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Device management & connectivity"
+title: "Device Management & Connectivity"
```
Comment (BeateRixen) [inline content/sector/device_management/_index.md:2]:
```suggestion
title: "Device management & connectivity"
```

File: content/streaming-analytics/analytics-builder-bundle/getting-started.md [hunk 20]
```diff
@@ -113,11 +113,11 @@ A card for the newly added model is shown on the **Models** tab of the model man
 1. Click the drop-down menu on the card which currently shows **Draft** and select **Production**.
 2. Click the toggle button on the card which currently shows **Inactive**. This changes the state to **Active**.
 
-#### Step 9: Go to the Device management and view the measurements {#step-9-go-to-the-device-management-and-view-the-measurements}
+#### Step 9: Go to the Device Management and view the measurements {#step-9-go-to-the-device-management-and-view-the-measurements}
```
Comment (BeateRixen) [inline content/streaming-analytics/analytics-builder-bundle/getting-started.md:116]:
```suggestion
#### Step 9: Go to the Device Management application and view the measurements {#step-9-go-to-the-device-management-and-view-the-measurements}
```

File: content/streaming-analytics/introduction-analytics.md [hunk 21]
```diff
@@ -9,7 +9,7 @@ aliases:
...
+Sensors result in `Measurement` or `Event` objects in {{< product-c8y-iot >}}, and devices can receive `Operation` objects created within the {{< product-c8y-iot >}} platform. All of these objects \(`Measurement`, `Event`, `Operation`\) will be associated with a single device in the {{< product-c8y-iot >}} platform. A device may have multiple types of measurement associated with it, and the types of measurements each device supports may be the same as other devices or different to other devices. Once devices are connected to {{< product-c8y-iot >}}, information about these devices is stored in the {{< product-c8y-iot >}} inventory. These are visible in the Device Management application, which can also be used to view `Measurement`, `Event` or `Operation` objects associated with that device. See [Device Management](/section/device_management/) for more information.
```
Comment (BeateRixen) [inline content/streaming-analytics/introduction-analytics.md:12]:
```suggestion
Sensors result in `Measurement` or `Event` objects in {{< product-c8y-iot >}}, and devices can receive `Operation` objects created within the {{< product-c8y-iot >}} platform. All of these objects \(`Measurement`, `Event`, `Operation`\) will be associated with a single device in the {{< product-c8y-iot >}} platform. A device may have multiple types of measurement associated with it, and the types of measurements each device supports may be the same as other devices or different to other devices. Once devices are connected to {{< product-c8y-iot >}}, information about these devices is stored in the {{< product-c8y-iot >}} inventory. These are visible in the Device Management application, which can also be used to view `Measurement`, `Event` or `Operation` objects associated with that device. See [Device management & connectivity](/section/device_management/) for more information.
```

File: data/component.json [hunk 22]
```diff
@@ -86,7 +86,7 @@
       "product_area": [
         {
           "value": "product_area-m1iHjqikD",
-          "label": "Device management & connectivity"
+          "label": "Device Management & Connectivity"
```
Comment (BeateRixen) [inline data/component.json:89]:
```suggestion
          "label": "Device management & connectivity"
```

File: data/component.json [hunk 23]
```diff
@@ -96,7 +96,7 @@
...
```
Comment (BeateRixen) [inline data/component.json:99]:
```suggestion
          "label": "Device management & connectivity"
```

File: data/component.json [hunk 24]
```diff
@@ -106,7 +106,7 @@
...
```
Comment (BeateRixen) [inline data/component.json:109]:
```suggestion
          "label": "Device management & connectivity"
```

Comment (BeateRixen) [reply data/component.json:109]: please revert all these changes in the json and yml files.

**Comment by BeateRixen** [comment]
@dafkk The capitalization only applies to the "Device Management application". In general, device management is not capitalized. Also, the headings in the documentation and change logs are called "Device management & connectivity" (not capitalized). I have done one general change directly in your branch (was much quicker than adding comments everywhere) and left quite some comments where you need to revert or modify changes.

File: content/concepts/interfacing-devices-bundle/support-of-developing-agents.md [hunk 25]
```diff
@@ -9,6 +9,6 @@ sector:
 
 {{< product-c8y-iot >}} supports agent development on three different levels:
 
-* It supports open source agents and drivers, for example [{{< product-c8y-iot >}} Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) or [thin-edge.io](https://thin-edge.github.io/thin-edge.io/).
+* It supports open source agents and drivers, for example [{{< product-c8y-iot >}} Device management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) or [thin-edge.io](https://thin-edge.github.io/thin-edge.io/).
```
Comment (BeateRixen) [inline content/concepts/interfacing-devices-bundle/support-of-developing-agents.md:12]:
```suggestion
* It supports open source agents and drivers, for example [{{< product-c8y-iot >}} Device Management Reference Agent](https://github.com/SoftwareAG/cumulocity-devicemanagement-agent) or [thin-edge.io](https://thin-edge.github.io/thin-edge.io/).
```

Comment (BeateRixen) [reply content/concepts/interfacing-devices-bundle/support-of-developing-agents.md:12]: This is a proper name which uses capitalization.

File: content/standard-tenant/managing-permissions-bundle/improving-performance.md [hunk 26]
```diff
@@ -10,9 +10,9 @@ The {{< product-c8y-iot >}} platform provides optimized UI performance for users
 
 The performance of the following UI pages is improved:
 * In the device details view, the tabs **Info**, **Measurements**, **Alarms**, **Events** and **Control**.
-* Pages with aggregated alarm views from multiple devices, if the number of alarms in the system is low, for example, Cockpit > Home dashboard, Cockpit > Alarms and Device management > Home.
-* Pages with aggregated events from multiple devices, if the number of events is low, for example, Device management > Events.
-* Pages with aggregated operations from multiple devices, if the number of operations is low, for example, Device management > Decie control > Single operations.
+* Pages with aggregated alarm views from multiple devices, if the number of alarms in the system is low, for example, Cockpit > Home dashboard, Cockpit > Alarms and Device management & connectivity > Home.
+* Pages with aggregated events from multiple devices, if the number of events is low, for example, Device management & connectivity > Events.
+* Pages with aggregated operations from multiple devices, if the number of operations is low, for example, Device management & connectivity > Decie control > Single operations.
```
Comment (BeateRixen) [inline content/standard-tenant/managing-permissions-bundle/improving-performance.md:15]:
```suggestion
* Pages with aggregated operations from multiple devices, if the number of operations is low, for example, Device management & connectivity > Device control > Single operations.
```

**Comment by BeateRixen** [comment]
@dafkk Has this change already been performed in the UI? We should only merge this PR if the UI has been adapted accordingly.

**Comment by dafkk** [comment]
> @dafkk Has this change already been performed in the UI? We should only merge this PR if the UI has been adapted accordingly.

Just has been merged - https://github.com/Cumulocity-IoT/cumulocity-ui/pull/6991

**Review by bozhidardedov — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**